web/src/narrow_title.ts

```diff
@@ -114,7 +114,7 @@ export function update_unread_counts(counts: FullUnreadCountsData): void {
     redraw_title();
 }
 
-export function update_narrow_title(filter: Filter): void {
+export function update_narrow_title(filter?: Filter): void {
```

Comment:
`compute_narrow_title` has optional `Filter`, so this was just a mistake before

Reply:
I don't see why it's relevant that `compute_narrow_filter` accepts `undefined`; the question should be whether the callers can call it with `undefined`, which they of course can.